language.go

```diff
@@ -156,6 +156,7 @@ var Exts = map[string]string{
 	"ipynb":       "Jupyter Notebook",
 	"jai":         "JAI",
 	"java":        "Java",
+	"jsp":         "Jsp",
```

Comment:
```suggestion
	"jsp":         "JSP",
```

Reply:
ok, I have changed the `Jsp` to an upper case